python/pyspark/sql/connect/session.py

```diff
@@ -489,10 +495,6 @@ def sparkContext(self) -> Any:
     def streams(self) -> Any:
         raise NotImplementedError("streams() is not implemented.")
 
-    @property
-    def readStream(self) -> Any:
-        raise NotImplementedError("readStream() is not implemented.")
-
```

Comment:
We are keeping the file names, class names, file paths etc similar between pyspark & connect. Thought it would be nice to keep the relative position of API methods also the same. E.g. `readStream()` follows `read()`.